### ISSUE TITLE
Trigger emit example

### ISSUE THREAD

**Issue by iivalchev** [onyx-platform/onyx-examples#9]
It would be great to have an example of a trigger that emits segments when fired. Think the aggregation example could be extended to have that.

**Comment by neverfox** [comment]
I'm really interested in this. But perhaps an explanation here would help too: when it says they're emitted downstream, how does one control that, e.g. via flow conditions? Are they emitted as segments of the task the trigger is related to? Something else?

**Comment by lbradstreet** [comment]
Yes, they are subject to flow conditions. Hopefully I’ll get around to the example soon (though we’d love to merge a PR from you). 

If you wish to excuse the regular segments, you can use the new onyx/type :reduce.
